pkg/kyverno/test/test_command.go

```diff
@@ -456,3 +451,37 @@ func printTestResult(resps map[string]report.PolicyReportResult, testResults []T
 	printer.Print(table)
 	return nil
 }
+
+func setInStoreContext(mutatedPolicies []*v1.ClusterPolicy, variables map[string]string) map[string]string {
```

Comment:
as `setInStoreContext` is same function used for apply command, move it in common folder

Reply:
sure